ci/priv-integration.sh

```diff
@@ -106,6 +108,19 @@ img_commit2=$(ostree --repo=${repo} rev-parse ostree/container/image/${imgref})
 test "${img_commit}" = "${img_commit2}"
 echo "ok deploy derived container identical revs"
 
+ostree-ext-cli container image deploy --sysroot "${sysroot}" \
```

Comment:
I think it'd be a bit better to do e.g.
```
skopeo copy ${derived_img} ${derived_img_dir}
```

here (*instead* of the above `systemd-run skopeo copy`), and then we can do the deployment...